pom.xml

```diff
@@ -844,11 +844,11 @@
             <version>2.2.2</version>
         </dependency>
         -->
+        <!-- https://mvnrepository.com/artifact/org.netbeans.external/jemmy-2.3.1.1 -->
         <dependency>
-            <groupId>org.netbeans</groupId>
-            <artifactId>jemmy</artifactId>
-            <version>2.2.7.5</version>
-            <scope>test</scope>
```

Comment:
Please do not change the scope. Removing the test scope implies that jemmy is needed for JMRI to run, not just for our unit testing (unless that has changed).

Reply:
I'll fix that, that was unintentional.